File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/vorkath/Teleport.java
package net.runelite.client.plugins.microbot.vorkath;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum Teleport {
    VARROCK_TAB("Varrock teleport", "break"),
    CRAFTING_CAPE("Crafting Cape", "teleport");


    ;private final String itemName;
    private final String action;

    @Override
    public String toString()
    {
        return itemName;
    }
}

File: runelite-client/src/main/java/net/runelite/client/plugins/jrPlugins/autoVorkath/enums/TELEPORT.kt
/*
 * Copyright (c) 2024. By Jrod7938
 *
 */
package net.runelite.client.plugins.jrPlugins.autoVorkath.enums

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class TELEPORT(private val teleportName: String, private val action: String) {
    CONSTRUCT_CAPE_T("Construct. cape(t)", "Tele to POH"),
    CONSTRUCT_CAPE("Construct. cape", "Tele to POH"),
    HOUSE_TAB("Teleport to house", "Break");

    override fun toString(): String = teleportName

    fun action() = action

}
